pkg/controller/kccmanager/kccmanager.go

```diff
@@ -100,7 +101,9 @@ type Config struct {
 // This serves as the entry point for the in-cluster main and the Borg service main. Any changes made should be done
 // with care.
 func New(ctx context.Context, restConfig *rest.Config, cfg Config) (manager.Manager, error) {
+	krmtotf.SetUserAgentForTerraformProvider()
```

Comment:
Also looks unrelated to the MCL integration (?)

Reply:
+1